raven/raven_ai/doctype/raven_ai_function/raven_ai_function.py

```diff
@@ -208,6 +208,28 @@ def prepare_function_params(self):
 			}
 		elif self.type == "Custom Function":
 			params = json.loads(self.params)
+		elif self.type == "Get List":
+			params = {
+				"type": "object",
+				"properties": {
+					"filters": {
+						"type": "object",
+						"description": "Filters to apply when retrieving the list",
```

Comment:
Wouldn't this need more details like "additional_properties" and the variable names?

Reply:
Yes, we are currently working on this and testing different variations. We're exploring how best to handle additional_properties and variable names to ensure flexibility and accuracy. Thank you for the suggestion!